index.js

```diff
@@ -1 +1 @@
-module.exports = require('./lib/Logger.js');
+module.exports = { Logger: require('./lib/Logger.js') };
```

Comment:
cf #75, we didn't export an object, nor a default field, which made the ES6 import fizzle.